### ISSUE TITLE
ibm-label doesn't support the use of "for" as an input, which is an accessibility violation

### ISSUE THREAD

**Issue by ozatici** [IBM/carbon-components-angular#1784]
"for" is used for labels to attach them to the input that they describe. This is an accessibility standard. ibm-label doesn't support the use of for as an input, so it needs to be added to mitigate accessibility violations.

**Comment by zvonimirfras** [comment]
Closed in #1785
